custom_components/econet300/binary_sensor.py

```diff
@@ -31,34 +33,69 @@ class EconetBinarySensorEntityDescription(BinarySensorEntityDescription):
         key="pumpCWUWorks",
         name="Water pump",
         icon="mdi:pump",
+        icon_off="mdi:pump-off",
         device_class=BinarySensorDeviceClass.RUNNING
     ),
     EconetBinarySensorEntityDescription(
         availability_key="pumpCirculation",
         key="pumpCirculationWorks",
         name="Circulation pump",
         icon="mdi:pump",
+        icon_off="mdi:pump-off",
         device_class=BinarySensorDeviceClass.RUNNING
     ),
     EconetBinarySensorEntityDescription(
         availability_key="pumpFireplace",
         key="pumpFireplaceWorks",
         name="Fireplace pump",
         icon="mdi:pump",
+        icon_off="mdi:pump-off",
         device_class=BinarySensorDeviceClass.RUNNING
     ),
     EconetBinarySensorEntityDescription(
         availability_key="pumpSolar",
         key="pumpSolarWorks",
         name="Solar pump",
         icon="mdi:pump",
+        icon_off="mdi:pump-off",
+        device_class=BinarySensorDeviceClass.RUNNING
+    ),
+    EconetBinarySensorEntityDescription(
+        availability_key="pumpCO",
+        key="pumpCOWorks",
+        name="pumpCO",
+        icon="mdi:pump",
+        icon_off="mdi:pump-off",
         device_class=BinarySensorDeviceClass.RUNNING
     ),
     EconetBinarySensorEntityDescription(
         availability_key="lighter",
         key="lighterWorks",
         name="Lighter",
         icon="mdi:fire",
+        icon_off="mdi:fire-off",
+        device_class=BinarySensorDeviceClass.RUNNING
+    ),
+    EconetBinarySensorEntityDescription(
+        availability_key="feeder",
+        key="feederWorks",
+        name="Feeder",
+        icon="mdi:screw-lag",
+        device_class=BinarySensorDeviceClass.RUNNING
+    ),
+    EconetBinarySensorEntityDescription(
+        availability_key="fan2Exhaust",
+        key="fanWorks",
+        name="Feeder",
+        icon="mdi:screw-lag",
+        device_class=BinarySensorDeviceClass.RUNNING
+    ),
+        EconetBinarySensorEntityDescription(
+        availability_key="fan",
+        key="fan2ExhaustWorks",
+        name="Feeder",
+        icon="mdi:fan",
```

Comment:
my bad I see my mistake with key when I make copy paste, sorry

Reply:
EconetBinarySensorEntityDescription(
        availability_key="fan",
        key="fanWorks",
        name="Fan", 
        icon="mdi:fan",
        device_class=BinarySensorDeviceClass.RUNNING
    ),

exapmle know in my case:
fan:  false
fanWorks : false


EconetBinarySensorEntityDescription(
        availability_key="fan2Exhaust",
        key="fan2ExhaustWorks", 
        name="Fan2",
        icon="mdi:fan",
        icon_off = "mdi:fan-off",
        device_class=BinarySensorDeviceClass.RUNNING
    )


exapmle know in my case:
fan2Exhaust: false
fan2ExhaustWorks : false